### PR TITLE
Added async/await to fix firefox issue + other things

### PR DESCRIPTION
**Pull request by zapdos26**
Fixes Firefox issue of not adding calls/notes.
Prevent notes from being empty;
Removed window reload upon adding note/call;
Added form reset on adding a note/call;
Added refresh of notes on adding a call;
Added await to getNotes in deleteCall();
Added `</div>` that was missing in the admin.html

Currently the reset of the addCall causes the validator to run (this occurs even with reloading the page). This will need to be looked into for Firefox.

### CODE REVIEW

File: public/js/admin.js [hunk 1]
```diff
@@ -25,29 +25,15 @@ async function getNotes() {
...
     const note = document.querySelector('#add-a-note').value;
-    fetch('/note/create', {
-        method: 'POST',
-        headers: {
-            'Content-Type': 'application/json'
-        },
-        redirect: 'follow',
-        body: JSON.stringify({ note })
-    });
+    await postServer('/note/create',note);
```
Comment (ddbruce) [inline public/js/admin.js]: I'm not sure if you ran this through ESLint or Prettier or some other program that does styling, but there should be a space after the `,` towards the end of this line.  Further, I'd ask that you call this function `postToServer` so it's a little more semantic.

Comment (ddbruce) [reply public/js/admin.js]: Also, this is a good place to do `const result = await postToServer[...]` and check if `result` passed or failed.

Comment (ddbruce) [reply public/js/admin.js]: ^ you don't need to do this, but it's good practice. Please, though, add that space after the comma :)

Comment (zapdos26) [reply public/js/admin.js]: Resolved

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "headsup",
-  "version": "2.0.1",
+  "version": "2.0.2",
```
Comment (ddbruce) [inline package.json]: Please modify this to fall in line with the change made in e04222f

Comment (ddbruce) [reply package.json]: (Hint: `1.1.2`)

**Comment by ddbruce** [comment]
Fixes #15

**Pull request merged**